frontend/package.json

```
@@ -25,7 +25,8 @@
     "vue2-dropzone": "3.6.0",
     "vuefire": "1.4.5",
     "vuetify": "1.5.18",
-    "vuex": "3.1.1"
+    "vuex": "3.1.1",
+    "whatwg-fetch": "^3.0.0"
```

Pin the dependency version in order to avoid Renovate doing this afterwards.
```suggestion
    "whatwg-fetch": "3.0.0"
```